Library/Formula/znc.rb

```diff
@@ -19,19 +17,22 @@ class Znc < Formula
     sha1 "83597cccd275a3a4bf8fcc5d8dd5c9048403869a" => :mountain_lion
   end
 
-  option "enable-debug", "Compile ZNC with --enable-debug"
+  option "with-debug", "Compile ZNC with debug support"
+
+  deprecated_option "enable-debug" => "with-debug"
 
   depends_on "pkg-config" => :build
   depends_on "openssl"
+  depends_on "icu4c" => :recommended
```

Comment:
Let's make this optional to start with and add a `with-icu4c` description that says about adding charset support.